src/layout/workspace.rs

```diff
@@ -1077,6 +1086,20 @@ impl<W: LayoutElement> Workspace<W> {
         first.chain(rest)
     }
 
+    // location of the active tile in global visual coordinates
+    pub fn active_tile_visual_pos(&self) -> Point<i32, Logical> {
+        let output_loc = self
```

Comment:
This output loc should be added in the function on Monitor, which always has an output.

Reply:
fixed in commit [123c2f0](https://github.com/YaLTeR/niri/pull/245/commits/123c2f040e1891c7ce5d3d7c29d497f48da788cd)